install_config/install/deploy_router.adoc

```diff
@@ -1061,18 +1061,19 @@ $ oadm policy add-scc-to-user privileged -z router
 
 Now that the router use can create privileged containers, make the
 router deployment configuration use the power so that the router can
```

Comment:
This is long and confusing: "Now that the router use can create privileged containers, make the router deployment configuration use the power so that the router can set the iptables rules it needs."

Suggested edit: "The router user can now create privileged containers. Next, configure the router deployment to use the privilege so that the router can set the iptables rules it needs."

(I'd refrain from using "power" in this way)